pkg/cmd/admin/router/router.go

```diff
@@ -293,6 +299,15 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg *
 					{Type: dapi.DeploymentTriggerOnConfigChange},
 				},
 				Template: dapi.DeploymentTemplate{
+					Strategy: dapi.DeploymentStrategy{
```

Comment:
There aren't any plans to change the default to -10% right now. Doing so would be a major change in the default behavior which would need discussed and documented, etc.